src/language-css/printer-postcss.js

```diff
@@ -697,6 +697,10 @@ function genericPrint(path, options, print) {
 
         // Add `hardline` after inline comment (i.e. `// comment\n foo: bar;`)
         if (isInlineValueCommentNode(iNode)) {
+          if (parentNode.type === "value-paren_group") {
```

Comment:
Potential unsafe, `value-paren_group` can be used in functions and nested function, need more tests, I just feel that it breaks something

Reply:
Having a look at this now, I'm not sure what kind of tests would suffice. I'll add nested functions for now, but I'm not sure if I'm writing the kind of tests you're looking for.